matlab/src/matlab/+arrow/+array/UInt64Array.m

```diff
@@ -12,7 +12,6 @@
 % WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or
 % implied.  See the License for the specific language governing
 % permissions and limitations under the License.
-<<<<<<< HEAD
```

Comment:
Oh, sorry. I missed this.

Reply:
No problem! :)

We are investigating internally with the team at MathWorks who maintains the testing framework why this didn't throw an error in GitHub Actions.